etcdserver/api/v3rpc/watch.go

```diff
@@ -61,15 +62,25 @@ func (ws *watchServer) Watch(stream pb.Watch_WatchServer) error {
 func sendLoop(stream pb.Watch_WatchServer, watcher storage.Watcher, closec chan struct{}) {
 	for {
 		select {
-		case e, ok := <-watcher.Chan():
+		case evs, ok := <-watcher.Chan():
 			if !ok {
 				return
 			}
-			err := stream.Send(&pb.WatchResponse{Event: &e})
+
+			// TODO: evs is []storagepb.Event type
+			// either return []*storagepb.Event from storage package
+			// or define protocol buffer with []storagepb.Event.
```

Comment:
There seems to be no easy way to make protobuf return non-pointer type for non-repeated fields, which means we should change the channel to return `[]*storagepb.Event`. But this needs lots of code change. I can do later.